configs/config

```diff
@@ -22,6 +22,7 @@ BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
 BR2_LINUX_KERNEL_LZ4=y
 BR2_PACKAGE_RPI_FIRMWARE=y
 BR2_PACKAGE_RPI_FIRMWARE_X=y
+BR2_PACKAGE_RPI_USERLAND=y
```

Comment:
You're installing all of RPi userland (e.g. raspivid, raspistill, dtoverlay etc.) here. Is that what we want?

Reply:
I kinda want to keep raspistill and raspivid to do debugging for end-users anyways. Those are nice. 

All the others can go but since we can't patch the makefile it'd be more pain than it's worth to not install them. What do you think?

Reply:
Maybe TODO in another PR https://github.com/showmewebcam/showmewebcam/issues/75

Reply:
It's just that some userland tools break, as necessary libraries are not found.

Reply:
OK, I will clean it up later in issue #75